tests/storage/restricted_namespace_cloning/conftest.py

```diff
@@ -34,13 +37,13 @@
 
 
 @pytest.fixture(scope="module")
-def destination_ns(unprivileged_client):
-    yield from create_ns(name="restricted-namespace-destination-namespace", unprivileged_client=unprivileged_client)
+def destination_namespace(admin_client):
```

Comment:
I don't understand this change.

Reply:
It makes more sense that both source and destination namespaces are created by admin, and then certain permissions are granted to the unprivileged user